### PR TITLE
Rework pangram exercise docs

### PR DESCRIPTION
**Pull request by kytrinyx**
ChatGPT suggested the following real-world usages for pangrams:

A pangram is a sentence or phrase that contains all the letters of the alphabet at least once. While pangrams can be a fun linguistic exercise or a tool for testing typing skills, there are also some real-life applications for them:

1. Typography and graphic design: Pangrams are often used by designers and typographers to test out different typefaces and see how the letters look when combined in a sentence. A pangram can help to ensure that all the letters are well-formed and easy to read.
2. Cryptography: Pangrams can also be used in cryptography, where they may be used to create a code or cipher. By using all the letters of the alphabet, a pangram can help to ensure that the code is comprehensive and covers all possible letters and combinations.
3. Language learning: For those learning a new language, pangrams can be a helpful tool for practicing the alphabet and becoming familiar with the different letters and sounds. By memorizing a pangram, learners can also gain a better understanding of sentence structure and grammar.
4. Game and puzzle creation: Pangrams can be used to create word games, puzzles, and riddles. For example, players may be challenged to come up with their own pangram or to find all the words that can be made using the letters in a given pangram.

Overall, while pangrams may not have a specific practical purpose in and of themselves, they can be a useful tool for a variety of applications in different fields.

_Editor's note: I like that ChatGPT and I agree on the first one. I also think (3) is kind of interesting, though not particularly useful in our case since we're limiting ourselves to ASCII. (4) is pretty funny: _find all the words that can be made using the letters in a given pangram_.... where a pangram uses all the letters in the alphabet at least once. By definition. I guess if you have repeat letters that will open up new possibilities for words?_

-----

Here's what ChatGPT came up with for the story, using the prompt you came up with:

Story: Have you ever heard of a pangram? It's a special type of sentence that contains every letter of the alphabet at least once. Some people believe that a pangram can help you learn how to write or type faster, because you have to use all the letters. In fact, many typing tests use pangrams to evaluate typing speed and accuracy. But there's more to pangrams than just typing. They can be fun to create and even more fun to read. Some pangrams are so clever and witty that they're like little puzzles or riddles.

Instructions: Your task is to determine whether a given sentence is a pangram or not. A pangram should contain all the letters of the alphabet at least once, including both uppercase and lowercase letters. To check whether a sentence is a pangram, you can write a program that loops through each letter of the alphabet and checks whether it appears in the sentence. Remember that the alphabet consists of letters a to z, inclusive, and is case insensitive. If the sentence contains all the letters of the alphabet, print "True", otherwise print "False".

For example, if the input sentence is "The quick brown fox jumps over the lazy dog", your program should output "True". If the input sentence is "Hello, world!", your program should output "False", because it does not contain the letters b, j, k, q, v, x, or z.

_Editor's note: this was terrible. I'm going to try tweaking the prompt for the next one._

### CODE REVIEW

File: exercises/pangram/introduction.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+A company that designs fonts wants to improve their website.
+They'd like to automatically show a random sentence when someone is looking at a font.
+To give a sense of what the font looks like, the random sentences should use all the letters in the English alphabet.
+
+They're running a competition to get suggestions for sentences that they can use.
+Your friend is in charge of checking the submissions to see if they are valid.
+
+The competition went viral on social media, and your friend needs to check thousands of sentences.
+
+They've asked you to help out.
```
Comment (iHiD) [inline exercises/pangram/introduction.md:12]:
```suggestion
They're running a competition to get suggestions for sentences that they can use.
Your in charge of checking the submissions to see if they are valid.
```

File: exercises/pangram/introduction.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+A company that designs fonts wants to improve their website.
+They'd like to automatically show a random sentence when someone is looking at a font.
+To give a sense of what the font looks like, the random sentences should use all the letters in the English alphabet.
```
Comment (iHiD) [inline exercises/pangram/introduction.md:5]: Simplifying and shortening a bit.

```suggestion
You work for a company that sells fonts through their website.
They'd like to show a different sentence each time someone views a font on their website. 
To give a comprehensive sense the font, the random sentences should use **all** the letters in the English alphabet.
```

File: exercises/pangram/instructions.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to figure out if a sentence is a pangram.
+
+A pangram is a sentence using every letter of the alphabet at least once.
+It is case insensitive, so it doesn't matter if a letter is lower-case (e.g. `k`) or upper-case (e.g. `K`).
+
+For this exercise we only use the basic letters used in English, `a` to `z`.
```
Comment (iHiD) [inline exercises/pangram/instructions.md:8]:
```suggestion
For this exercise we only use the basic letters used in English: `a` to `z`.
```

**Review by iHiD — APPROVED** [review]
Let's go!

File: exercises/pangram/instructions.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to figure out if a sentence is a pangram.
+
+A pangram is a sentence using every letter of the alphabet at least once.
+It is case insensitive, so it doesn't matter if a letter is lower-case (e.g. `k`) or upper-case (e.g. `K`).
+
+For this exercise we only use the basic letters used in English: `a` to `z`.
```
Comment (ErikSchierboom) [inline exercises/pangram/instructions.md:8]: Nice and clear!

File: exercises/pangram/introduction.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Introduction
+
+You work for a company that sells fonts through their website.
+They'd like to show a different sentence each time someone views a font on their website.
+To give a comprehensive sense the font, the random sentences should use **all** the letters in the English alphabet.
```
Comment (ErikSchierboom) [inline exercises/pangram/introduction.md:5]:
```suggestion
To give a comprehensive sense of the font, the random sentences should use **all** the letters in the English alphabet.
```

File: exercises/pangram/introduction.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+They're running a competition to get suggestions for sentences that they can use.
+You're in charge of checking the submissions to see if they are valid.
```
Comment (ErikSchierboom) [inline exercises/pangram/introduction.md:8]: Great story!

**Review by ErikSchierboom — APPROVED** [review]

**Pull request closed without merging**